malware/plugx/match-known-module.yml

```
@@ -0,0 +1,63 @@
+rule:
+  meta:
+    name: match known module
+    namespace: malware/plugx
```

```suggestion
    namespace: malware-family/plugx
```

and change file path

---

Done 🙌